service/src/test/java/bio/terra/javatemplate/BaseSpringBootTest.java

```diff
@@ -0,0 +1,14 @@
+package bio.terra.javatemplate;
+
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.annotation.Rollback;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.transaction.annotation.Transactional;
+
+@SpringBootTest(
+    properties = {"DATABASE_NAME=javatemplate_db"},
+    classes = App.class)
```

Comment:
This should be unnecessary if `App.class` is in the correct location (I had it in the wrong place earlier and moved into the right place recently)